File: problems/80. Remove Duplicates from Sorted Array II/main.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        repeats = 2
        size = len(nums)

        if size <= repeats:
            return size

        k = repeats
        for i in range(repeats, size):
            if nums[i] != nums[k - repeats]:
                nums[k] = nums[i]
                k += 1
        return k

File: problems/80. Remove Duplicates from Sorted Array II/main.go
func removeDuplicates(nums []int) int {
	repeats := 2
	size := len(nums)

	if size <= repeats {
		return size
	}

	k := repeats
	for i := repeats; i < size; i++ {
		if nums[i] != nums[k-repeats] {
			nums[k] = nums[i]
			k++
		}
	}
	return k
}
